geopandas/geoseries.py

```diff
@@ -1223,7 +1229,13 @@ def to_json(self, **kwargs) -> str:
         --------
         GeoSeries.to_file : write GeoSeries to file
         """
-        return json.dumps(self.__geo_interface__, **kwargs)
+        from geopandas import GeoDataFrame
+
+        return json.dumps(
```

Comment:
we should also still pass **kwargs through to json.dumps.

Reply:
Thanks Matt, I missed that